.github/workflows/docs.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: Docsite CI
+on:
+  pull_request:
+    paths:
+      - 'docs/**'
+      - '.github/workflows/docs.yml'
+jobs:
+  docsite-build:
+    name: docsite test build
+    runs-on: ubuntu-latest
```

Comment:
How about using Fedora? One of my goals is to start using Fedora in all our tests. It'd be cool to start here.

Reply:
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners

The choices are Ubuntu, Windows, or Mac.

Reply:
nuts. I could swear I noticed Fedora in another repo somewhere and it's kind of been on my to-do list to start looking at the switch. sorry for the noise.